src/tutorials/code/node/constellation/1.js

```diff
@@ -0,0 +1,6 @@
+const Carina = require('carina').Carina;
+const ws = require('ws');
+
+Carina.WebSocket = ws;
```

Comment:
that would probably bloat the tutorial a lot. Tutorial is designed for node environments too